> AssertionError: ["In file '''/home/runner/work/rosdistro/rosdistro/noetic/distribution.yaml''': Could not validate source entry for repo qb_device with error [[[Pull Request builds only supported on GitHub right now. Cannot do pull request against bitbucket.org]]]"] is not false

I'm checking why the "Nosetests" are failing, I'm not really sure what's the issue

--- a/noetic/distribution.yaml
+++ b/noetic/distribution.yaml
@@ -5967,7 +5967,6 @@ repositories:
       url: https://bitbucket.org/qbrobotics/qbdevice-ros-release.git
       version: 2.2.1-1
     source:
-      test_pull_requests: true
       type: git
       url: https://bitbucket.org/qbrobotics/qbdevice-ros.git
       version: production-noetic

